BCStudentSoftwareDevTeam/celts
# Merge Proposal Management with Service Learning under Admin Log

This merges the Proposal Management Page with Service Learning Page. Now admin can manage Service Learning Faculty and Service Learning Course on the page `/manageServiceLearning.`

Before: 
<img width="1440" alt="Before" src="https://user-images.githubusercontent.com/59736834/166280942-32616e08-7345-4eb7-b0f4-da3d4e7b505a.png">
After:
<img width="1425" alt="After" src="https://user-images.githubusercontent.com/59736834/166281167-73fb20c8-b449-4c6a-8d03-d3b74f7f03df.png">

Test: 
- navigate to `/manageServiceLearning`
- try to change the term by using drop down menu under Service Learning Course Proposal Management and it should work fine.
- `Add Course` Button should also be working fine.
